package.json

```diff
@@ -18,6 +18,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test-server": "cross-env NODE_ENV=test nodemon server.js",
+    "test-server-db": "npm run docker-db-stop && npm run docker-db-start && npm run test-server",
```

Comment:
I think you can rewrite `test-server` and instead of run `server.js` we have `app.js`, where we running a server for testing.

Reply:
fixed

Reply:
fixed